ledger-service/http-json/src/main/scala/com/digitalasset/http/ContractsFetch.scala

```diff
@@ -109,7 +108,7 @@ private class ContractsFetch(
               )
             )
         },
-        fconn.pure(a),
+        within(bb),
```

Comment:
This changes the behaviour of the function. Check the comment

> /** run `within` repeatedly after fetchAndPersist until the latter is
>     * consistent before and after `within`
>     */
are we ok with that?